scripts/commands/derivatives/msgs.sh

```diff
@@ -12,8 +12,11 @@ $BINARY tx derivatives burn-lpt 1 ubtc --from=$USER1 $conf | jq .raw_log | sed '
 $BINARY tx derivatives burn-lpt 1 ubtc --from=$USER1 $conf | jq .raw_log | sed 's/\\n/\n/g'
 
 # open-position perpetual-futures
-$BINARY tx derivatives open-position perpetual-futures 100ubtc ubtc uusdc long --from=$USER1 $conf | jq .raw_log | sed 's/\\n/\n/g'
-$BINARY tx derivatives open-position perpetual-futures 100uusdc ubtc uusdc short --from=$USER1 $conf | jq .raw_log | sed 's/\\n/\n/g'
+$BINARY tx derivatives open-position perpetual-futures 100ubtc ubtc uusd long --from=$USER1 $conf | jq .raw_log | sed 's/\\n/\n/g'
```

Comment:
plz change "uusd" to "uusdc" after the change of the markets values in genesis.json.
plz refer init.sh file comment

Reply:
uusdc does't work.
uusd is working

Reply:
with this modification (https://github.com/UnUniFi/chain/pull/388#discussion_r1112528525), it should work.

Reply:
No, it does not work.
I have tried that.

Reply:
sorry, needed one more modification.
https://github.com/UnUniFi/chain/pull/388#discussion_r1113750136

Reply:
It's working